pkg/deploy/api/types.go

```diff
@@ -292,6 +292,10 @@ type DeploymentConfigSpec struct {
 	// Replicas is the number of desired replicas.
 	Replicas int
 
+	// RevisionHistoryLimit is the number of old ReplicationControllers to retain to allow for rollbacks.
```

Comment:
> It's ok to have the description match upstream.

Upstream uses "ReplicaSets". Ours talks about "ReplicationControllers". In sight of having both work as one object, I am not sure what's better here. "Deployments" meaning a different thing between here and upstream is another thing.